src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewKernel.php

```diff
@@ -33,11 +34,13 @@ public function registerContainerConfiguration(LoaderInterface $loader)
     {
         parent::registerContainerConfiguration($loader);
 
-        if (in_array($this->getEnvironment(), ['dev', 'test'])) {
-            $loader->load(
-                implode(DIRECTORY_SEPARATOR, [__DIR__, '..', '..', 'Resources', 'config', 'config_preview_dev.yml'])
-            );
-        }
+        $loader->load(function(ContainerBuilder $container) use ($loader) {
```

Comment:
`// disable web_profiler toolbar in preview if the web_profiler is enabled`

Reply:
Comment added